invisible_cities/reco/pmaps_functions.py

```diff
@@ -113,3 +113,19 @@ def scan_s2si_map(S2Si):
         for nsipm, E in sipm.items():
             ene = np.sum(E)
             print('SiPM number = {}, total energy = {}'.format(nsipm, ene))
+
+
+def width(times, to_mus=False):
+    """
+    Compute peak width.
+    """
+    w = (np.max(times) - np.min(times)) if np.any(times) else 0.
```

Comment:
I don't quite get this line. You check if `times` has some not null element, if not you return zero. But it all array elements are zero, `max` and `min` will actually be 0, so the `if` part seems irrelevant.

Also, it seems that the times for a peak are always ordered, so getting the maximum and the minimum is reduced to just taking the first and the last element.